weave

```diff
@@ -1444,6 +1505,8 @@ case "$COMMAND" in
         echo $RES
         ;;
     attach)
+        extra_hosts=
+        rewrite_hosts=
```

Comment:
Yes, you are right, I usually use lower-case names for local vars... I'll change it